src/model/Stroke.cpp

```diff
@@ -355,7 +318,7 @@ void Stroke::rotate(double x0, double y0, double xo, double yo, double th)
 		p.y += y0;
 
 		p.x += xo-offset;	//center it
-		p.y += yo-offset;		
+		p.y += yo - offset;
```

Comment:
yes, thats the problem with clang-format, if i change it to the previous format, the check will fail. I could change this line also. Or we ignore it, since the format will be applied with the PR freeze.